docs/elements.md

```diff
@@ -1703,7 +1703,15 @@ An overlay is pre-defined as a "overlay area" with a static size. By default, el
 
 ### `pl-external-grader-variables` element
 
-Displays variables that are given to the student, or expected for the student to define in externally-graded questions. The list of variables should be stored in `data['params']` and has the following format:
+Displays variables that are given to the student, or expected for the student to define in externally-graded Python questions.
+In particular, **we advise against using this element for any non-Python externally graded question**, since the way this element
+stores parameters is specifically for use with the Python autograder.
+
+The list of variables can be defined in in `data['params']` or the question HTML itself (but not both!). If defined in the
```

Comment:
Nit for consistency with https://github.com/PrairieLearn/PrairieLearn/pull/6794:

```suggestion
The list of variables can be defined in in `data["params"]` or the question HTML itself (but not both!). If defined in the
```